tests/functional/test_basic.py

```diff
@@ -21,7 +21,7 @@
 from PyInstaller.utils.tests import importorskip, skipif_win, skipif_winorosx, skipif_notwin
 
 # Directory with data for some tests.
-_DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
+from tests.functional.data_dir import DATA_DIR
```

Comment:
Similar here. Could you move the `DATA_DIR` variable to `conftest.py` or `PyInstaller/utils/tests.py`?